app/views/pizzas/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1 style='margin-top:100px'> Record a new Pizzeria in NYC</h1>
+
+<div class='form-container'>
+
+  <%= @pizzeria.name %></label><br />
+  <%=  @pizzeria.address %></label><br /> 
+
```

Comment:
You have label closing tags here but no opening tags. I think you could just omit the closing tags altogether though.